CODEOWNERS

```diff
@@ -56,3 +56,5 @@ Makefile*                                                        @istio/wg-test-
 /tools/packaging/common/envoy_bootstrap.json                  @istio/wg-networking-maintainers
 /tools/istio-iptables/                                           @istio/wg-networking-maintainers
 /tools/istio-clean-iptables/                                     @istio/wg-networking-maintainers
+manifests/charts/global.yaml                                     @costinm @ostromart @sdake
+**/values.yaml                                                   @costinm @ostromart @sdake
```

Comment:
I am not sure this will expand as you want. This makes sense, although listing each file separately may be better.

Reply:
according to the docs it will, but i'll test it with a pr.